tzatziki-spring-jpa/src/main/java/com/decathlon/tzatziki/steps/SpringJPASteps.java

```diff
@@ -115,11 +117,27 @@ public void before() {
         }
     }
 
+    private Pair<String, String> getTableSchemaAndName(Class<?> clazz) {
+        return Optional.ofNullable(clazz.getAnnotation(PersistenceUtil.getPersistenceClass("Table")))
+                .map(tableAnnotation -> {
+                    String tableName = (String) Optional.ofNullable(AnnotationUtils.getValue((Annotation) tableAnnotation, "name"))
+                            .orElse(Optional.ofNullable(clazz.getAnnotation(org.springframework.data.relational.core.mapping.Table.class))
+                                    .map(org.springframework.data.relational.core.mapping.Table::value).orElse(null)
+                            );
+                    String schemaName = (String) AnnotationUtils.getValue((Annotation) tableAnnotation, "schema");
+                    return Pair.of(schemaName, tableName);
+                }).orElse(Pair.of(null, null));
```

Comment:
Est-ce qu'on ne devrait pas throw une exception si on n'en trouve pas ? il n'y a pas un risque de nullPointerException par la suite ?

Reply:
In the method getTableNameWithSchema we are checking if the schema is null, to only return table name is this case that can be null. 
The worst case is that we could try to TRUNCATE null, but it is already the current behavior if I'm not wrong.